src/plasoscaffolder/bll/mappings/mapping_helper.py

```diff
@@ -15,18 +14,21 @@ def __init__(self, template_path: str):
       template_path (str): the path to the template directory
     """
     super().__init__()
-    self.template_path = template_path
+    self.__template_path = template_path
 
-  def _get_template_environment(self) -> Environment:
+  @property
+  def _GetTemplateEnvironment(self) -> jinja2.Environment:
```

Comment:
Maybe use:
Get => Create

Seeing that the method creates a new template environment